Include/internal/pycore_runtime.h

```diff
@@ -192,6 +236,14 @@ typedef struct _Py_DebugOffsets {
         uint64_t size;
         uint64_t collecting;
     } gc;
+
+    struct _gen_object {
+        uint64_t size;
+        uint64_t gi_name;
+        uint64_t gi_iframe;
+        uint64_t gi_task;
+        uint64_t gi_frame_state;
```

Comment:
This is an implementation detail, and may not exist in future versions of CPython